src/Illuminate/Auth/Passwords/DatabaseTokenRepository.php

```diff
@@ -172,6 +172,28 @@ public function createNewToken()
     }
 
     /**
+     * Get and validate token data from database.
+     *
+     * @param  string  $token
+     * @return bool
+     */
+    public function validateToken($token)
+    {
+        $token = $this->getTable()->where('token', $token)->first();
+
+        if (empty($token)) {
+            return false;
+        }
+
+        $expirationDate = Carbon::parse($token->created_at)->addSeconds($this->expires);
```

Comment:
add a newline after this, then this looks good - we don't yet have a fixer for styleci to check that rule ;)